charts/parca/templates/clusterrole.yaml

```diff
@@ -1,20 +1,23 @@
+{{- if and .Values.server.enabled .Values.server.rbac.create }}
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  labels:
-    {{- include "parca.labels" . | nindent 4 }}
   name: {{ include "parca.serviceAccountName" . }}
+  labels:
+    app: {{ include "parca.serviceAccountName" . }}
+{{- include "parca.labels" . | nindent 4 }}
 rules:
-- apiGroups:
-  - ""
+- apiGroups: [""]
   resources:
+  - nodes
+  - nodes/metrics
+  - services
+  - endpoints
   - pods
-  verbs:
-  - list
-  - watch
+  verbs: ["get", "list", "watch"]
 - apiGroups:
-  - ""
+  - "networking.k8s.io"
```

Comment:
while possible (because we're re-using the Prometheus discovery), I can't come up with a useful scenario where one would want to be scraping ingresses (and services for that matter)

Reply:
You have a point for the ingress section, i did remove it. 
However, for services it still makes sense, for me at least. You might have a STS or pod behind a service for which you can use DNS, rather than an IP for scraping.

As an alternative, i can remove them from this section, and add them in values, using something like [kube-prometheus](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml#L2943) toghether with the section explained below.